handlers/bind.go

```diff
@@ -87,6 +87,16 @@ func (h APIHandler) Bind(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	if binding.AlreadyExists {
```

Comment:
We think it makes more sense to check if the instance already exists before checking if the response is async. Whether the response is async or synchronous, we want to return 200 if the instance already exists. 

There would need to be a new test to be sure that if the broker response is asyn and says that the instance already exists that the returned response is 200.